streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -1514,8 +1521,6 @@ public void shouldOnlyCommitRevokedStandbyTaskAndPropagatePrepareCommitException
 
     @Test
     public void shouldSuspendAllRevokedActiveTasksAndPropagateSuspendException() {
-        setUpTaskManager(StreamThread.ProcessingMode.EXACTLY_ONCE_ALPHA);
```

Comment:
Not sure why this test sets up the TM for EOS? Does anyone know?

Reply:
cc @abbccdda